README.md

```diff
@@ -268,6 +283,18 @@ Supported functions:
 
 Full documentation: [visualization](https://texthero.org/docs/api-visualization)
 
+<h2 align="center">Pandas Typing</h2>
+
+What is pandas typing? Each category of functions, which are defined above, accepts a different type of pandas series or a pandas dataframe. Those are explained here. These types will in general improve the efficiency of the different algorithms as they are especially designed for those.
+
+For the first step you don't need to worry too much about them, because, if you use a typical texthero pipeline, the functions will have the right input and output types. The *typical* hero pipeline will:
```

Comment:
Maybe "You usually don't need to worry about them, because, if you use a typical texthero pipeline, the different types are seamlessly integrated across all modules". The *typical* hero pipeline will:"

Reply:
ok, 🐤